bundle/analyze/superset/superset-init.sh

```diff
@@ -17,10 +17,7 @@
 #
 set -e
 
-#
-# Always install local overrides first
-#
-/app/docker/docker-bootstrap.sh
+export FLASK_APP=superset
```

Comment:
is this required?

Reply:
@DouweM yeah in https://superset.apache.org/docs/installation/installing-superset-from-scratch#installing-and-initializing-superset it tells you to export it otherwise flask will fail to start as part of the startup command they provide `superset run -p 8088 --with-threads --reload --debugger` which I customized a bit. Again we could potentially swap this out for gunicorn though like in https://github.com/meltano/files-superset/pull/12#discussion_r889576901.